community-operators/planetscale/planetscale-operator.v0.1.8.clusterserviceversion.yaml

```diff
@@ -128,8 +130,8 @@ spec:
sIGg/MDzyHN0Oov5tXg1QJfyGEiGcClOF5e2E01bLP+joJAYWQASCE6CuLurqNC6i+ i+1GwwVVFvCqzzadC8AA/t7Pmmo9qquy+t/0zO4G4a81/kIIke7941y1HCBv6K0pR0DIABBC9O1l CWBNx99lPa0I0GY2/rWabFPE93BAdfpwbRYMgfldOHzZheR+rjIFtMZfCCGS4z2UM8jWqaSisoBC BoAQYuOwkSOQU78AY5sAm1wRYJsHI21UBKBa42+rrP43YXLTUZZLlKgSQgiRwAUIJoDNT5ZiSawL IQNAiO3UR0sk+GtlLXUz0jjd8caMAuCqD9UzAZZqMnbfVJ9fad99MgFMWD5TzuDuc57JLUc5AZNH ykgthBBi5We5m0O2qglwyQtMUQBCBoAQYiOE/yaYAK0eb6PnvuH5ACIYECubAMs0X83quzncfcFz dNszP/IYQqk/IYQQHXgHAeVckQBCyAAQYtvegZsSgqwX9lr91WaERCu5CBJ9l4wNdacntzwHLznK o3CcmvEXQohu4so1qq80muxXCBkAQog1VZfZtDdUioSAifuo15egzwe/6rGf9zlTJfgrYXLbc/hS KOm3MAWEEEJ0GA//f/buPF7Wq67z/Wc9T9Xe+ww5GYAWxOlqK9rq9dJqi6J9b6vQYquN3BYVcLwO 7YjIJHMYgkgIEOapZZJBVEhAmQVCS5gMcxgCgSRnzDk5w67aNdez1v1jPSecIEnOHmv6vF+v/SLs s3ft2r+qc6p+32et30rjvELr5pNbtvr1zURABgCSJtb3fO1y/80ccTaNL4LTfCrAFM8C2HBttmgp /yRmEWzFc7goIUXor+Yj/YZrieQVf0matQyAWG8H2NA8AFcByABA0vR1/nWTFbauE3IWwARDgHkp 9ISPBdzIt58O0VKCfivRPxUZdvMKACf7S9KMhgApbwfYymNZDQFkACBpMo1/OLuvm5tXqWleBbCV d2ArbmcrVgHM6jukdf7c0439zUv9T+Wp/rHKf8cKXxUlabZDgFj/e7/eEOA2Xk8MAWQAIGlH+5sN DbXZid5t1rYC7MAr+MyuApjxWQC3dyJAKPKbwWoIw7VE70RkPODmfaIu95ekOQ0B7N4lAwBpZpqa LZxmO7MhwJQ/Rju3CmB9acgkVwFM1aDGAGUJ1aje438yL/UPoV4NEPynRpLmMgSoV3axni1dhgUy AJC04z1U2IKmZKdDgFnqwTc0qW59d2D7GuBtDgG2cDvCJIYR3mIVQICiPs6vczwP9xt1EwGv9kvS oohVfmkJxTpeXxwIKAMASTvWzDp4zFfcLa7XJLdJTCQEqAOeVEGvFemdgnH3jKn+XvGXpIV6PUzx jBVfWxACSAYAkjb/GrUVV/0XuEmfxoxgpnOLGbzzp4c9xQH0WolBOw/3O/3GL/iuTZIWMwSg3g7g PADJAECahhembWtM5mkWwLS/YG/VcsF5eWNyxu+x3asAQpE/qiEMWoneyUQ1SFDUyz5d7i9JSmeE AJIBgKRJ9EcUO/SDFiQEmMZZANsTApzFHd+KgYCbfSy3+/vryf3VEPqrkX4Lxr2UA4EGN0/3lyTp 5hAg1Rdezub1ydUCMgCQtCV94qwv959o8bY4BJjR33MjtzHJ2mzlzz490TmO6uF+rci4XwcCja/5 oZIknfm6EPNrCIYAMgCQtGPN/4x0XwtzLOCWdqfs4OTgGSj0Vg4ULPNtxREM2oneqUg1zFdzCl/J JElnKUW3AsgAQNI8Nv6TDAHmpJ+f6B2YhlUAU3AsYAj1Hv/BLRv/03+vPDlDkrRukbNfKeYqABkA SJqZxn8eu/DtuO2ZfXE/izs+6b38m/i7ExpQ9evhfqci40G+amPTL0najFTPiXG3mAwAJNn4z0Hf O5Ghd9tx89twnzY0+3CL7sdZ/ewzh/sdTwxaiXE/D/crfcWSJG1xCLB9L56SAYBk4z/FTfOW936L vGTOWQDrvounr+rHMXRPRAarMB4k9/hLkrbtpckTY2QAIGk+m/95bMqn/USAKZ4FMJHbOOP7z7yp 08P9qhEM24n+aqQa5M+F0vdlkiRJBgCSzf9s9LLb3YVvdQiwA6nBxFYBTFn4E4p6uN8wD/frn0pU gwT15+38JUk79iYHXN4vAwBJ63jRWIBu3mMBt+62tieYWN+tTuRhqX9o0ciNf7eVGKwmxoNEUXgs kyRpAtbzYmg4LQMAycZ/Zu//vIQA0z4QcEe3AqyvGDu5QuLm4/xG0DmWGLQioz43D/2TJEmSAYBk 4z9lIYC/6rSHAOu7je0OAW4e7jeCfjvRb+U9/inWw/28miJJmhZuA5ABgKS5a/zNDabiF92+m5+O QQahzFf9qwEM1hK9U5FqWP9ZgOCrjyTJNzmSAYAkX1QX5QV7Ynd7s6sAbuumQ27+bx7utxoZ96Eo 8ockSdPM/l8GANKiNqs25VvfuDoLYPM/YhsGAm7J71Pv5a+G0D8RGbRg3E+Ewj3+kqQZemsUIJkC yABAkmbhVXsbQoAZ/D03chsbDUhOH4EZx9A5GRm2YDRIedp/06ekJGlGX2YNAWQAIGkRm+Yt70E9 FnCHSrK9hQ5Fbv7jOO/x75/KS/1JXvGXJM3BeyNwP4AMACQZAkx9Lz/tqwB2dCvA+m7jbG4yFHmP fxxBfy3RP5kY94AQ8moA9/lLkuaC3b8MACRpR5r0qfvxM7t34GzG9Z/971Y08hX/fivRX02MOnUg 0PB9kiRpzt6qFIFU+eImAwBJNuVb0xvP2qkAO7AVYHtuZ30hwL9Z9Rhyk58q6J9I9NuJUTf/Udn4 mtuQJGmu3hsFksMAZAAgyRDA+sx7+UPIdyhFGKzmK/5VD2Ks9/jb8EuS5v1lv8ivg743kgGAJE17 bjCjswAmXfwQ6j3+Yxj28pF+o17+GSHkbQCSJC3MexxXAcgAQJJmJDmYwWMBJ3YiQMhX9lOEYRt6 rcSwk4/
```

Comment:
`replaces` should be the previous CSV's `metadata.name`. Setting this to `planetscale-operator.v0.1.7` should be fine

Reply:
Got it, thanks for the help. I’ll work on these and push again.

Reply:
Changes pushed. Also added to upstream.

Reply:
Made the necessary changes to the files for upstream.